src/phoenix/db/migrations/versions/cf03bd6bae1d_init.py

```diff
@@ -0,0 +1,94 @@
+"""init
+
+Revision ID: cf03bd6bae1d
+Revises:
+Create Date: 2024-04-03 19:41:48.871555
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "cf03bd6bae1d"
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    projects_table = op.create_table(
+        "projects",
+        sa.Column("id", sa.Integer, primary_key=True),
+        sa.Column("name", sa.String, nullable=False),
+        sa.Column("description", sa.String, nullable=True),
+        # TODO(mikeldking): is timezone=True necessary?
```

Comment:
Yea, it doesn't work with sqlite, because it just stores it as strings. I end up having to call `isofomat()` manually. Maybe this is useful for other vendors. Ultimately, having a timezone-aware python `datetime` should be the expectation.